tests/unit_test/app/exceptions/test_handlers.py

```
@@ -0,0 +1,186 @@
+    """
+    Tests for the UnauthorizedAccess subclass of DevDoxAPIException, via /boom/unauth.
+    """
+
+    def test_subclass_propagates_http_status(self, fastapi_permissive_client):
+        """Ensures that the overridden http_status = 401 is correctly returned."""
+        resp = fastapi_permissive_client.get("/boom/unauth")
+        assert resp.status_code == 401
+        assert resp.json()["status_code"] == 401
+        assert resp.json()["debug"]["exception"] == "UnauthorizedAccess"
+
+    def test_default_auth_message_included(self, fastapi_permissive_client):
```

Pylint (3.3.7)

[warning] Redefining name 'fastapi_permissive_client' from outer scope (line 61)

(W0621)